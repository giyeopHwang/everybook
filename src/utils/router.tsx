import { createBrowserRouter } from 'react-router-dom';

import Front from '@/components/front/Front.component';
import DashboardLayout from '@/components/dashboard/dashboard-layout/DashboardLayout.compoent';

export const router = createBrowserRouter([
  {
    path: '/',
    element: <Front />,
  },
  {
    path: 'dashboard',
    element: <DashboardLayout />,
    children: [
      {
        path: '',
        element: <div>가계부</div>,
      },
      {
        path: 'report',
        element: <div>보고서</div>,
      },
      {
        path: 'budget',
        element: <div>예산 관리</div>,
      },
      {
        path: 'account',
        element: <div>분류 관리</div>,
      },
    ],
  },
]);
