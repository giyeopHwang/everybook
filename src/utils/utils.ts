export const displayAmount = (
  amount: number,
  showAmount: boolean = true,
  placeholder: string = '*********'
) => {
  return showAmount ? amount.toLocaleString() : placeholder;
};
