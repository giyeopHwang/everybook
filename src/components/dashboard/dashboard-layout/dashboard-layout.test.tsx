import { renderWithWrapper } from '@/test/test.utils';
import DashboardLayout from './dashboard-layout.compoent';

describe('DashboardLayout', () => {
  it('renders DashboardLayout component', () => {
    const dashboardLayout = renderWithWrapper(<DashboardLayout />);
    expect(dashboardLayout).toMatchSnapshot();
  });
});
