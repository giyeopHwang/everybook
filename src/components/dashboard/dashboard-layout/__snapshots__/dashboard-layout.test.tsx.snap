// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`DashboardLayout > renders DashboardLayout component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="css-bq3f98"
      >
        <div
          class="css-5tru4f"
        >
          <a
            class="css-ad54sp"
            href="/dashboard"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              style="width: 1.5rem; height: 1.5rem;"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M2.25 18.75a60.07 60.07 0 0 1 15.797 2.101c.727.198 1.453-.342 1.453-1.096V18.75M3.75 4.5v.75A.75.75 0 0 1 3 6h-.75m0 0v-.375c0-.621.504-1.125 1.125-1.125H20.25M2.25 6v9m18-10.5v.75c0 .414.336.75.75.75h.75m-1.5-1.5h.375c.621 0 1.125.504 1.125 1.125v9.75c0 .621-.504 1.125-1.125 1.125h-.375m1.5-1.5H21a.75.75 0 0 0-.75.75v.75m0 0H3.75m0 0h-.375a1.125 1.125 0 0 1-1.125-1.125V15m1.5 1.5v-.75A.75.75 0 0 0 3 15h-.75M15 10.5a3 3 0 1 1-6 0 3 3 0 0 1 6 0Zm3 0h.008v.008H18V10.5Zm-12 0h.008v.008H6V10.5Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
            <span>
              모두의 가계부
            </span>
          </a>
          <nav
            class="css-1djzm1k"
          >
            <a
              class="css-grrdwo"
              href="/dashboard"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 1rem; height: 1rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="m16.862 4.487 1.687-1.688a1.875 1.875 0 1 1 2.652 2.652L10.582 16.07a4.5 4.5 0 0 1-1.897 1.13L6 18l.8-2.685a4.5 4.5 0 0 1 1.13-1.897l8.932-8.931Zm0 0L19.5 7.125M18 14v4.75A2.25 2.25 0 0 1 15.75 21H5.25A2.25 2.25 0 0 1 3 18.75V8.25A2.25 2.25 0 0 1 5.25 6H10"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <span>
                가계부
              </span>
            </a>
            <a
              class="css-grrdwo"
              href="/dashboard/report"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 1rem; height: 1rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M19.5 14.25v-2.625a3.375 3.375 0 0 0-3.375-3.375h-1.5A1.125 1.125 0 0 1 13.5 7.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H8.25M9 16.5v.75m3-3v3M15 12v5.25m-4.5-15H5.625c-.621 0-1.125.504-1.125 1.125v17.25c0 .621.504 1.125 1.125 1.125h12.75c.621 0 1.125-.504 1.125-1.125V11.25a9 9 0 0 0-9-9Z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <span>
                보고서
              </span>
            </a>
            <a
              class="css-grrdwo"
              href="/dashboard/budget"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 1rem; height: 1rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M10.5 6a7.5 7.5 0 1 0 7.5 7.5h-7.5V6Z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
                <path
                  d="M13.5 10.5H21A7.5 7.5 0 0 0 13.5 3v7.5Z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <span>
                예산 관리
              </span>
            </a>
            <a
              class="css-grrdwo"
              href="/dashboard/account"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 1rem; height: 1rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M21 12a2.25 2.25 0 0 0-2.25-2.25H15a3 3 0 1 1-6 0H5.25A2.25 2.25 0 0 0 3 12m18 0v6a2.25 2.25 0 0 1-2.25 2.25H5.25A2.25 2.25 0 0 1 3 18v-6m18 0V9M3 12V9m18 0a2.25 2.25 0 0 0-2.25-2.25H5.25A2.25 2.25 0 0 0 3 9m18 0V6a2.25 2.25 0 0 0-2.25-2.25H5.25A2.25 2.25 0 0 0 3 6v3"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <span>
                분류 관리
              </span>
            </a>
          </nav>
          <section
            class="css-14chwgd"
          >
            <header
              class="css-kympkc"
            >
              <div
                class="css-359auk"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 1rem; height: 1rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="m4.5 15.75 7.5-7.5 7.5 7.5"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h2
                  class="css-1bt39ed"
                >
                  이달의 자산현황
                </h2>
              </div>
            </header>
            <div
              class="css-7ttcrf"
            >
              <div
                class="css-135bpbl"
              >
                <header
                  class="css-kvjl0w"
                >
                  <div
                    class="css-vmk80s"
                  >
                    <svg
                      aria-hidden="true"
                      data-slot="icon"
                      fill="none"
                      stroke="currentColor"
                      stroke-width="1.5"
                      style="width: 0.9rem; height: 0.9rem;"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M3.75 9h16.5m-16.5 6.75h16.5"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    <h4
                      class="css-pqtvky"
                    >
                      순자산
                    </h4>
                  </div>
                  <span>
                    500,000,000
                  </span>
                </header>
                <section
                  class="css-1r2jysi"
                />
              </div>
              <div
                class="css-135bpbl"
              >
                <header
                  class="css-1r6e9a4"
                >
                  <div
                    class="css-vmk80s"
                  >
                    <svg
                      aria-hidden="true"
                      data-slot="icon"
                      fill="none"
                      stroke="currentColor"
                      stroke-width="1.5"
                      style="width: 0.9rem; height: 0.9rem;"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M12 4.5v15m7.5-7.5h-15"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    <h4
                      class="css-pqtvky"
                    >
                      자산
                    </h4>
                  </div>
                  <span>
                    500,000,000
                  </span>
                </header>
                <section
                  class="css-1r2jysi"
                >
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      현금잔액
                    </span>
                    <span>
                      100,000,000
                    </span>
                  </div>
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      투자자산
                    </span>
                    <span>
                      350,000,000
                    </span>
                  </div>
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      기타자산
                    </span>
                    <span>
                      50,000,000
                    </span>
                  </div>
                </section>
              </div>
              <div
                class="css-135bpbl"
              >
                <header
                  class="css-1i1w77g"
                >
                  <div
                    class="css-vmk80s"
                  >
                    <svg
                      aria-hidden="true"
                      data-slot="icon"
                      fill="none"
                      stroke="currentColor"
                      stroke-width="1.5"
                      style="width: 0.9rem; height: 0.9rem;"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M5 12h14"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    <h4
                      class="css-pqtvky"
                    >
                      부채
                    </h4>
                  </div>
                  <span>
                    100,000,000
                  </span>
                </header>
                <section
                  class="css-1r2jysi"
                >
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      대출
                    </span>
                    <span>
                      75,000,000
                    </span>
                  </div>
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      기타부채
                    </span>
                    <span>
                      25,000,000
                    </span>
                  </div>
                </section>
              </div>
            </div>
          </section>
          <section
            class="css-14chwgd"
          >
            <header
              class="css-kympkc"
            >
              <div
                class="css-359auk"
              >
                <svg
                  aria-hidden="true"
                  data-slot="icon"
                  fill="none"
                  stroke="currentColor"
                  stroke-width="1.5"
                  style="width: 1rem; height: 1rem;"
                  viewBox="0 0 24 24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="m4.5 15.75 7.5-7.5 7.5 7.5"
                    stroke-linecap="round"
                    stroke-linejoin="round"
                  />
                </svg>
                <h2
                  class="css-1bt39ed"
                >
                  이달의 가계
                </h2>
              </div>
              <h3
                class="css-1puzo0z"
              >
                2024.01.01 ~ 2024.01.31
              </h3>
            </header>
            <div
              class="css-7ttcrf"
            >
              <div
                class="css-135bpbl"
              >
                <header
                  class="css-kvjl0w"
                >
                  <div
                    class="css-vmk80s"
                  >
                    <svg
                      aria-hidden="true"
                      data-slot="icon"
                      fill="none"
                      stroke="currentColor"
                      stroke-width="1.5"
                      style="width: 0.9rem; height: 0.9rem;"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M3.75 9h16.5m-16.5 6.75h16.5"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    <h4
                      class="css-pqtvky"
                    >
                      수입 - 지출
                    </h4>
                  </div>
                  <span>
                    7,000,000
                  </span>
                </header>
                <section
                  class="css-1r2jysi"
                />
              </div>
              <div
                class="css-135bpbl"
              >
                <header
                  class="css-1r6e9a4"
                >
                  <div
                    class="css-vmk80s"
                  >
                    <svg
                      aria-hidden="true"
                      data-slot="icon"
                      fill="none"
                      stroke="currentColor"
                      stroke-width="1.5"
                      style="width: 0.9rem; height: 0.9rem;"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M12 4.5v15m7.5-7.5h-15"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    <h4
                      class="css-pqtvky"
                    >
                      수입
                    </h4>
                  </div>
                  <span>
                    10,000,000
                  </span>
                </header>
                <section
                  class="css-1r2jysi"
                >
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      이달의 수입
                    </span>
                    <span>
                      8,000,000
                    </span>
                  </div>
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      전월이월
                    </span>
                    <span>
                      2,000,000
                    </span>
                  </div>
                </section>
              </div>
              <div
                class="css-135bpbl"
              >
                <header
                  class="css-1i1w77g"
                >
                  <div
                    class="css-vmk80s"
                  >
                    <svg
                      aria-hidden="true"
                      data-slot="icon"
                      fill="none"
                      stroke="currentColor"
                      stroke-width="1.5"
                      style="width: 0.9rem; height: 0.9rem;"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M5 12h14"
                        stroke-linecap="round"
                        stroke-linejoin="round"
                      />
                    </svg>
                    <h4
                      class="css-pqtvky"
                    >
                      지출
                    </h4>
                  </div>
                  <span>
                    3,000,000
                  </span>
                </header>
                <section
                  class="css-1r2jysi"
                >
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      현금지출
                    </span>
                    <span>
                      1,000,000
                    </span>
                  </div>
                  <div
                    class="css-uf1ume"
                  >
                    <span>
                      └ 
                      카드지출
                    </span>
                    <span>
                      2,000,000
                    </span>
                  </div>
                </section>
              </div>
            </div>
          </section>
        </div>
        <div
          class="css-1rfq2pv"
        >
          <header
            class="css-5whm5h"
          >
            <button
              class="css-iwi5cf"
              data-testid="nav-bar-collapse-button"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M15.75 19.5 8.25 12l7.5-7.5"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
            </button>
            <div
              class="css-jh5a00"
            >
              <h2
                class="css-uzu4z8"
              >
                가계부
              </h2>
              <button
                class="css-wieti"
              >
                <span>
                  금액감추기
                </span>
                <span
                  class="css-101dbv1"
                >
                  OFF
                </span>
              </button>
            </div>
          </header>
        </div>
      </div>
      <footer
        class="css-1psyufs"
      >
        <div
          class="css-jh5a00"
        >
          <span>
            이달의예산
          </span>
          <div
            class="css-1sz5t5j"
          >
            <div
              class="css-1dv1ys6"
            />
          </div>
          <span
            class="css-md2hjx"
          >
            3,000,000
          </span>
          <span>
            /
          </span>
          <span>
            5,000,000
          </span>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <div
      class="css-bq3f98"
    >
      <div
        class="css-5tru4f"
      >
        <a
          class="css-ad54sp"
          href="/dashboard"
        >
          <svg
            aria-hidden="true"
            data-slot="icon"
            fill="none"
            stroke="currentColor"
            stroke-width="1.5"
            style="width: 1.5rem; height: 1.5rem;"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M2.25 18.75a60.07 60.07 0 0 1 15.797 2.101c.727.198 1.453-.342 1.453-1.096V18.75M3.75 4.5v.75A.75.75 0 0 1 3 6h-.75m0 0v-.375c0-.621.504-1.125 1.125-1.125H20.25M2.25 6v9m18-10.5v.75c0 .414.336.75.75.75h.75m-1.5-1.5h.375c.621 0 1.125.504 1.125 1.125v9.75c0 .621-.504 1.125-1.125 1.125h-.375m1.5-1.5H21a.75.75 0 0 0-.75.75v.75m0 0H3.75m0 0h-.375a1.125 1.125 0 0 1-1.125-1.125V15m1.5 1.5v-.75A.75.75 0 0 0 3 15h-.75M15 10.5a3 3 0 1 1-6 0 3 3 0 0 1 6 0Zm3 0h.008v.008H18V10.5Zm-12 0h.008v.008H6V10.5Z"
              stroke-linecap="round"
              stroke-linejoin="round"
            />
          </svg>
          <span>
            모두의 가계부
          </span>
        </a>
        <nav
          class="css-1djzm1k"
        >
          <a
            class="css-grrdwo"
            href="/dashboard"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              style="width: 1rem; height: 1rem;"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m16.862 4.487 1.687-1.688a1.875 1.875 0 1 1 2.652 2.652L10.582 16.07a4.5 4.5 0 0 1-1.897 1.13L6 18l.8-2.685a4.5 4.5 0 0 1 1.13-1.897l8.932-8.931Zm0 0L19.5 7.125M18 14v4.75A2.25 2.25 0 0 1 15.75 21H5.25A2.25 2.25 0 0 1 3 18.75V8.25A2.25 2.25 0 0 1 5.25 6H10"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
            <span>
              가계부
            </span>
          </a>
          <a
            class="css-grrdwo"
            href="/dashboard/report"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              style="width: 1rem; height: 1rem;"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M19.5 14.25v-2.625a3.375 3.375 0 0 0-3.375-3.375h-1.5A1.125 1.125 0 0 1 13.5 7.125v-1.5a3.375 3.375 0 0 0-3.375-3.375H8.25M9 16.5v.75m3-3v3M15 12v5.25m-4.5-15H5.625c-.621 0-1.125.504-1.125 1.125v17.25c0 .621.504 1.125 1.125 1.125h12.75c.621 0 1.125-.504 1.125-1.125V11.25a9 9 0 0 0-9-9Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
            <span>
              보고서
            </span>
          </a>
          <a
            class="css-grrdwo"
            href="/dashboard/budget"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              style="width: 1rem; height: 1rem;"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M10.5 6a7.5 7.5 0 1 0 7.5 7.5h-7.5V6Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
              <path
                d="M13.5 10.5H21A7.5 7.5 0 0 0 13.5 3v7.5Z"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
            <span>
              예산 관리
            </span>
          </a>
          <a
            class="css-grrdwo"
            href="/dashboard/account"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              style="width: 1rem; height: 1rem;"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M21 12a2.25 2.25 0 0 0-2.25-2.25H15a3 3 0 1 1-6 0H5.25A2.25 2.25 0 0 0 3 12m18 0v6a2.25 2.25 0 0 1-2.25 2.25H5.25A2.25 2.25 0 0 1 3 18v-6m18 0V9M3 12V9m18 0a2.25 2.25 0 0 0-2.25-2.25H5.25A2.25 2.25 0 0 0 3 9m18 0V6a2.25 2.25 0 0 0-2.25-2.25H5.25A2.25 2.25 0 0 0 3 6v3"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
            <span>
              분류 관리
            </span>
          </a>
        </nav>
        <section
          class="css-14chwgd"
        >
          <header
            class="css-kympkc"
          >
            <div
              class="css-359auk"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 1rem; height: 1rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="m4.5 15.75 7.5-7.5 7.5 7.5"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h2
                class="css-1bt39ed"
              >
                이달의 자산현황
              </h2>
            </div>
          </header>
          <div
            class="css-7ttcrf"
          >
            <div
              class="css-135bpbl"
            >
              <header
                class="css-kvjl0w"
              >
                <div
                  class="css-vmk80s"
                >
                  <svg
                    aria-hidden="true"
                    data-slot="icon"
                    fill="none"
                    stroke="currentColor"
                    stroke-width="1.5"
                    style="width: 0.9rem; height: 0.9rem;"
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M3.75 9h16.5m-16.5 6.75h16.5"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                    />
                  </svg>
                  <h4
                    class="css-pqtvky"
                  >
                    순자산
                  </h4>
                </div>
                <span>
                  500,000,000
                </span>
              </header>
              <section
                class="css-1r2jysi"
              />
            </div>
            <div
              class="css-135bpbl"
            >
              <header
                class="css-1r6e9a4"
              >
                <div
                  class="css-vmk80s"
                >
                  <svg
                    aria-hidden="true"
                    data-slot="icon"
                    fill="none"
                    stroke="currentColor"
                    stroke-width="1.5"
                    style="width: 0.9rem; height: 0.9rem;"
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M12 4.5v15m7.5-7.5h-15"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                    />
                  </svg>
                  <h4
                    class="css-pqtvky"
                  >
                    자산
                  </h4>
                </div>
                <span>
                  500,000,000
                </span>
              </header>
              <section
                class="css-1r2jysi"
              >
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    현금잔액
                  </span>
                  <span>
                    100,000,000
                  </span>
                </div>
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    투자자산
                  </span>
                  <span>
                    350,000,000
                  </span>
                </div>
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    기타자산
                  </span>
                  <span>
                    50,000,000
                  </span>
                </div>
              </section>
            </div>
            <div
              class="css-135bpbl"
            >
              <header
                class="css-1i1w77g"
              >
                <div
                  class="css-vmk80s"
                >
                  <svg
                    aria-hidden="true"
                    data-slot="icon"
                    fill="none"
                    stroke="currentColor"
                    stroke-width="1.5"
                    style="width: 0.9rem; height: 0.9rem;"
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M5 12h14"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                    />
                  </svg>
                  <h4
                    class="css-pqtvky"
                  >
                    부채
                  </h4>
                </div>
                <span>
                  100,000,000
                </span>
              </header>
              <section
                class="css-1r2jysi"
              >
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    대출
                  </span>
                  <span>
                    75,000,000
                  </span>
                </div>
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    기타부채
                  </span>
                  <span>
                    25,000,000
                  </span>
                </div>
              </section>
            </div>
          </div>
        </section>
        <section
          class="css-14chwgd"
        >
          <header
            class="css-kympkc"
          >
            <div
              class="css-359auk"
            >
              <svg
                aria-hidden="true"
                data-slot="icon"
                fill="none"
                stroke="currentColor"
                stroke-width="1.5"
                style="width: 1rem; height: 1rem;"
                viewBox="0 0 24 24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="m4.5 15.75 7.5-7.5 7.5 7.5"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                />
              </svg>
              <h2
                class="css-1bt39ed"
              >
                이달의 가계
              </h2>
            </div>
            <h3
              class="css-1puzo0z"
            >
              2024.01.01 ~ 2024.01.31
            </h3>
          </header>
          <div
            class="css-7ttcrf"
          >
            <div
              class="css-135bpbl"
            >
              <header
                class="css-kvjl0w"
              >
                <div
                  class="css-vmk80s"
                >
                  <svg
                    aria-hidden="true"
                    data-slot="icon"
                    fill="none"
                    stroke="currentColor"
                    stroke-width="1.5"
                    style="width: 0.9rem; height: 0.9rem;"
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M3.75 9h16.5m-16.5 6.75h16.5"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                    />
                  </svg>
                  <h4
                    class="css-pqtvky"
                  >
                    수입 - 지출
                  </h4>
                </div>
                <span>
                  7,000,000
                </span>
              </header>
              <section
                class="css-1r2jysi"
              />
            </div>
            <div
              class="css-135bpbl"
            >
              <header
                class="css-1r6e9a4"
              >
                <div
                  class="css-vmk80s"
                >
                  <svg
                    aria-hidden="true"
                    data-slot="icon"
                    fill="none"
                    stroke="currentColor"
                    stroke-width="1.5"
                    style="width: 0.9rem; height: 0.9rem;"
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M12 4.5v15m7.5-7.5h-15"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                    />
                  </svg>
                  <h4
                    class="css-pqtvky"
                  >
                    수입
                  </h4>
                </div>
                <span>
                  10,000,000
                </span>
              </header>
              <section
                class="css-1r2jysi"
              >
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    이달의 수입
                  </span>
                  <span>
                    8,000,000
                  </span>
                </div>
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    전월이월
                  </span>
                  <span>
                    2,000,000
                  </span>
                </div>
              </section>
            </div>
            <div
              class="css-135bpbl"
            >
              <header
                class="css-1i1w77g"
              >
                <div
                  class="css-vmk80s"
                >
                  <svg
                    aria-hidden="true"
                    data-slot="icon"
                    fill="none"
                    stroke="currentColor"
                    stroke-width="1.5"
                    style="width: 0.9rem; height: 0.9rem;"
                    viewBox="0 0 24 24"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M5 12h14"
                      stroke-linecap="round"
                      stroke-linejoin="round"
                    />
                  </svg>
                  <h4
                    class="css-pqtvky"
                  >
                    지출
                  </h4>
                </div>
                <span>
                  3,000,000
                </span>
              </header>
              <section
                class="css-1r2jysi"
              >
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    현금지출
                  </span>
                  <span>
                    1,000,000
                  </span>
                </div>
                <div
                  class="css-uf1ume"
                >
                  <span>
                    └ 
                    카드지출
                  </span>
                  <span>
                    2,000,000
                  </span>
                </div>
              </section>
            </div>
          </div>
        </section>
      </div>
      <div
        class="css-1rfq2pv"
      >
        <header
          class="css-5whm5h"
        >
          <button
            class="css-iwi5cf"
            data-testid="nav-bar-collapse-button"
          >
            <svg
              aria-hidden="true"
              data-slot="icon"
              fill="none"
              stroke="currentColor"
              stroke-width="1.5"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M15.75 19.5 8.25 12l7.5-7.5"
                stroke-linecap="round"
                stroke-linejoin="round"
              />
            </svg>
          </button>
          <div
            class="css-jh5a00"
          >
            <h2
              class="css-uzu4z8"
            >
              가계부
            </h2>
            <button
              class="css-wieti"
            >
              <span>
                금액감추기
              </span>
              <span
                class="css-101dbv1"
              >
                OFF
              </span>
            </button>
          </div>
        </header>
      </div>
    </div>
    <footer
      class="css-1psyufs"
    >
      <div
        class="css-jh5a00"
      >
        <span>
          이달의예산
        </span>
        <div
          class="css-1sz5t5j"
        >
          <div
            class="css-1dv1ys6"
          />
        </div>
        <span
          class="css-md2hjx"
        >
          3,000,000
        </span>
        <span>
          /
        </span>
        <span>
          5,000,000
        </span>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;
